homework-1/task-1.2/index.js

```diff
@@ -0,0 +1,96 @@
+const fs = require('fs');
+const csv = require('csvtojson');
+const path = require('path');
+const { pipeline } = require('stream');
+
+const csvFolder = './csv';
+const resultFolder = './result-whole';
+const resultLineByLineFolder = './result-line-by-line';
+
+chechResultFolder(resultFolder);
+chechResultFolder(resultLineByLineFolder);
+
+fs.readdir(csvFolder, function (err, files) {
+    if (err) {
+        return console.log(`Unable to scan directory: ${err}`);
+    } 
+    files.forEach(function (file) {
+        console.log(`Converting file ${file}`); 
+        let resultFile = path.join(resultFolder, path.basename(file, path.extname(file)) + '.txt')
```

Comment:
just a quick note on this one: you could do without passing the 2nd argument to `path.basename` thus freeing yourself from concatenating back the file format.
path.basename would return you `file + extension name`

Reply:
I want to change file extension from `.csv` (which is the initial extension for file) to `.txt` here. Without the second argument it will return `.csv`